singularity-eos/CMakeLists.txt

```diff
@@ -59,6 +59,8 @@ register_headers(
 
 if (SINGULARITY_BUILD_CLOSURE)
   register_headers(closure/mixed_cell_models.hpp)
+  register_headers(eos/singularity_eos.hpp)
+  register_srcs(eos/singularity_eos.cpp)
```

Comment:
what necessitated this change?

Reply:
@Yurlungur ?

Reply:
I didn't change this, @rbberger did, but I think he just moved these registration calls from the fortran if statement to the closure if statement. Since this is C++ code technically, not fortran code.

Reply:
this will blow up compile times in RIOT. While it is c++ source, they are really only needed in w/in the context of defining fortran interfaces and for EAP.

Reply:
oh this is a good catch. Thank you! Yes let's move it back into the fortran. I can submit a PR. @rbberger will this be ok?